resources/views/emails/training/waiting_list_non_home_removal.blade.php

```diff
@@ -0,0 +1,9 @@
+@extends('emails.messages.post')
+
+@section('body')
+
+<p>As you have changed your VATSIM Division away from the UK Division you are no longer eligible to receive ATC training in the UK. You were on one of the waiting lists for ATC training in the UK and as such as you are no longer a member of the UK Division you have been removed from the waiting list.</p>
```

Comment:
```suggestion
<p>As you have changed your VATSIM Division away from the UK Division you are no longer eligible to receive ATC training in the UK. You were on one of the waiting lists for ATC training in the UK and, as you are no longer a member of the UK Division, you have been removed from this waiting list.</p>
```

Reply:
That'll teach me from copying and pasting from the trello ticket :D